src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/Dependencies/Models/AnalyzerDependencyModel.cs

```diff
@@ -10,7 +10,9 @@ namespace Microsoft.VisualStudio.ProjectSystem.Tree.Dependencies.Models
 {
     internal class AnalyzerDependencyModel : DependencyModel
     {
-        private static readonly DependencyFlagCache s_flagCache = new DependencyFlagCache(add: DependencyTreeFlags.AnalyzerDependency);
+        private static readonly DependencyFlagCache s_flagCache = new DependencyFlagCache(
+            resolved: DependencyTreeFlags.AnalyzerDependency + DependencyTreeFlags.SupportsBrowse,
+            unresolved: DependencyTreeFlags.AnalyzerDependency + DependencyTreeFlags.SupportsBrowse);
```

Comment:
Do the commands associated with browsing work when the target does not exist?

Reply:
They attempt to open the parent folder, but no op after that. There's no flag to indicate missing on disk, and I'm hesistent to check the disk on QueryStatus.

I went to show an error message but its at the wrong layer and there was a significant refactoring due to the debug profile dialog using it and that being part of its _public API_, so I punted for another day.